pkg/controller/tuned/tuned_controller.go

```diff
@@ -278,15 +294,44 @@ func createCustomResource(mgr manager.Manager) error {
 	return nil
 }
 
+func addOwnerReference(meta *metav1.ObjectMeta, tuned *tunedv1alpha1.Tuned) []metav1.OwnerReference {
```

Comment:
Your desired functionality of `addOwnerReference()` over the built-in `SetControllerReference()` is to be able to manipulate two owner references (one controller and a secondary)? I would have just wrapped `SetOwnerReferences`.